apps/main-app/src/components/dataTable/AnnotatedEntitiesPopupCuration.js

```diff
@@ -84,7 +85,7 @@ function AnnotatedEntitiesPopupCuration(props) {
                     <td><ExperimentalConditionCellCuration conditions={entity.conditionRelations}/></td>
                     <td><ExperimentalConditionCellCuration conditions={entity.conditionModifiers}/></td>
                     <td><StrainBackground strainBackground={entity.sgdStrainBackground}/></td>
-                    <td><GeneticSex geneticSex={entity.geneticSex}/></td>
+                    <td>{entity.geneticSex && GeneticSex(geneticSex=entity.geneticSex)}</td>
```

Comment:
You can just add a entity.geneticSex? and it will not fail but then you need to check it in the component.

Reply:
Plus the error is not here... its in the component. `startsWith` doesn't exist.

Reply:
added entity.geneticSex?

Reply:
The error is not coming from the AnnotatedEntitiesPopupCuration.js file: its coming from the GeneticSex component.